Scripts/gen_Primary.sh

```diff
@@ -195,17 +184,28 @@ if [[ "${FLAT}" == "FlatMuDaughter" ]]; then
   echo physics.producers.generate.endMom: ${ENDMOM}        >> primary.fcl
 fi
 
-#
-# now generate the fcl
-#
+if [[ -n $SETUP ]]; then
+  echo "Using user-provided setup $SETUP"
+else
+  SETUP=/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/${PRIMARY_CAMPAIGN}/setup.sh
+fi
 
-generate_fcl --dsconf=${PRIMARY_CAMPAIGN} --dsowner=${OWNER} --run-number=${RUN} --description=${PRIMARY} --events-per-job=${EVENTS} --njobs=${JOBS} \
-  --embed primary.fcl --auxinput=1:physics.filters.${resampler}.fileNames:Stops.txt
-for dirname in 000 001 002 003 004 005 006 007 008 009; do
-  if test -d $dirname; then
-    echo "found dir $dirname"
-    rm -rf ${PRIMARY}\_$dirname
-    mv $dirname ${PRIMARY}\_$dirname
-    echo "moving $dirname to ${PRIMARY}_${dirname}"
-  fi
-done
+if [[ "$PROD" = true ]]; then
+    rm cnf.*.tar
```

Comment:
I thought 'prod' was to trigger sam registration, do you now plan to organize that some other way?
Since prod option is commented out this I don't think this line can actually do anything.
Is there a reason not to rm cnf.*.tar even without 'prod==true'? The following command won't work if there is a pre-existing file.

Reply:
This will only work on grid.
I don't want to delete user's file by default